ironfish/src/mining/poolDatabase/database.ts

```diff
@@ -71,6 +71,17 @@ export class PoolDatabase {
     return result.count
   }
 
+  async getAddressSharesCountForPayout(publicAddress: string): Promise<number> {
+    const result = await this.db.get<{ count: number }>(
+      'SELECT COUNT(*) AS count from share WHERE payoutId IS NULL and publicAddress = ?',
+      publicAddress,
+    )
+    if (result == null) {
+      return 0
+    }
```

Comment:
Can the sqlite driver really return null here?

Reply:
I think `if (result === undefined)` is more correct here. The sqlite driver will return undefined if the result set is empty. That shouldn't happen with this query in practice, but I suppose it's possible.

Reply:
Ok, if it's possible then I'm curious what the conditions are for a count(*) query to be empty. I can't get it to happen even when the table is empty.